app/views/errors/internal_server_error.html.erb

```diff
@@ -1,8 +1,16 @@
-<main role="main" class="govuk-main-wrapper" id="main-content">
```

Comment:
Why did you have to remove `main` element? Was it because these should be partials and therefore should be using the main application layout?

Reply:
Because the application template was already inserting a `<main>`, which means you'd end up with a duplicate `<main>`.

Reply:
Thank you for clarifying